jose/Jose.mli

```diff
@@ -66,7 +69,7 @@ module Jwk : sig
     [rsa_to_pub_pem t] takes a public JWK and returns a result public PEM string or a message of what went wrong.
     *)
 
-    val oct_of_string : string -> oct
+    val oct_of_string : ?use:string -> string -> oct
```

Comment:
tweak the doc for the new `?use` arg

Reply:
I added some docs for `use`, I also opened another PR against this branch to add JWE where this will be more useful and changed the implementation of `use` from a string to a poly variant.